src/main/webapp/app/admin/privacy-statement/privacy-statement-update/privacy-statement-update.component.html

```diff
@@ -0,0 +1,32 @@
+<div class="privacy-statement-container">
+    <div>
+        <h2>
+            <span id="privacy-statement-update-heading" jhiTranslate="artemisApp.privacyStatement.createOrEditLabel" data-cy="privacyStatementPageHeading"
+                >Create or edit privacy statement</span
+            >
+        </h2>
+
+        <jhi-markdown-editor
+            [markdown]="privacyStatement.text"
+            (markdownChange)="checkUnsavedChanges($event)"
+            [resizableMinHeight]="minHeight"
+            [resizableMaxHeight]="maxHeight"
+            [enableResize]="true"
+            [minHeightEditor]="700"
+            class="markdown-editor"
+        ></jhi-markdown-editor>
+        <div class="btn-below-editor">
+            <jhi-button
+                id="update-privacy-statement-btn"
+                (click)="updatePrivacyStatement()"
+                [isLoading]="isSaving"
+                [disabled]="!markdownEditor?.markdown || markdownEditor?.markdown?.trim() === ''"
+                [title]="'entity.action.save'"
+                [icon]="faSave"
+                [ngStyle]="{ 'marginRight.px': 10 }"
```

Comment:
Since a scss file exists for this component: Can this style definition be moved there?